.github/workflows/ci.yml

```diff
@@ -737,6 +738,11 @@ jobs:
             - suite: suite-delta-lake-databricks104
               ignore exclusion if: >-
                 ${{ secrets.DATABRICKS_TOKEN != '' }}
+            - suite: suite-delta-lake-databricks113
+              config: hdp3
+            - suite: suite-delta-lake-databricks113
+              ignore exclusion if: >-
+                ${{ secrets.DATABRICKS_TOKEN != '' }}
```

Comment:
Why is this two entries?

Reply:
It's under `exclude` section. These two entries are required to ignore hdp3 always and then enable another one when the secret exists in my understanding.

Reply:
LGTM. Note it adds quite a lot of time to the total CI runtime:
![obraz](https://user-images.githubusercontent.com/795177/201721731-bf319dfc-3183-4525-81bc-3fb9ec3a148e.png)

Do we need to test all LTSes?